### PR TITLE
 [7] Develop a Contact Form Page using HTML 

### PR DESCRIPTION
**Pull request by Dawodurahmonn**
kazeem
## Task: `Name of given task`
- `Changes you made to given task`

### CODE REVIEW

File: submissions/DawoduRahmon/HTML/product-page.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/DawoduRahmon/HTML/product-page.html:1]: Remove this file from your PR @Dawodurahmonn You have index.html already.

File: submissions/DawoduRahmon/HTML/product page.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/DawoduRahmon/HTML/product page.html:1]: Remove this file from your PR @Dawodurahmonn You have index.html already.

File: submissions/DawoduRahmon/htmlForms/contact.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<head>
+	<meta charset="UTF-8">
+    <meta name="viewport" content="width=div, initial-scale=1.0">
+    <title>Contact-Form</title>
+</head>
+<body>
```
Comment (ToluAkin) [inline submissions/DawoduRahmon/htmlForms/contact.html:7]: You did not close this
@Dawodurahmonn

File: submissions/DawoduRahmon/htmlForms/contact.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<a href="../html/index.html" target="blank">Home</a>
+<div>
+	<h1>Rilogad Limited Nigeria</h1>
+</div>
+<form>
+	<label>First Name <input type="text"/></label>
```
Comment (ToluAkin) [inline submissions/DawoduRahmon/htmlForms/contact.html:13]: This is the right format
<label>First Name</label>
<input type="text">
Kindly update @Dawodurahmonn

File: submissions/DawoduRahmon/htmlForms/contact.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+	<br/>
+	<br/>
+	<label>Last Name <input type="text"/></label>
+	<br/>
+	<br/>
+	<label>Email <input type="text" name="username" size="40"value/></label>
```
Comment (ToluAkin) [inline submissions/DawoduRahmon/htmlForms/contact.html:19]: No need to add `size` and `value` @Dawodurahmonn

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/DawoduRahmon/HTML/product-page.html (hunk 1 above)
Comment (Dawodurahmonn) [reply submissions/DawoduRahmon/HTML/product-page.html:1]: ma, pls ma i only have one index.html in my folder.
kindly put me through on what to do here ma.

File: submissions/DawoduRahmon/htmlForms/contact.html (hunk 4 above)
Comment (Dawodurahmonn) [reply submissions/DawoduRahmon/htmlForms/contact.html:13]: pls what should i update on these very one ma?

File: submissions/DawoduRahmon/htmlForms/contact.html (hunk 3 above)
Comment (Dawodurahmonn) [reply submissions/DawoduRahmon/htmlForms/contact.html:7]: Do you mean the body tag ma?

File: submissions/DawoduRahmon/HTML/product-page.html (hunk 1 above)
Comment (ToluAkin) [reply submissions/DawoduRahmon/HTML/product-page.html:1]: Exactly, you have a product page already which is your index.html. These extra product pages you added are not needed.
 Still on that same branch, update your changes.
Add your changes (git add .).
Commit your changes (git commit  -m  "commit message").
Finally, Push your change (git push origin branch_name)
@Dawodurahmonn

File: submissions/DawoduRahmon/htmlForms/contact.html (hunk 3 above)
Comment (ToluAkin) [reply submissions/DawoduRahmon/htmlForms/contact.html:7]: Yes @Dawodurahmonn . You did not close the body tag.

File: submissions/DawoduRahmon/htmlForms/contact.html (hunk 4 above)
Comment (ToluAkin) [reply submissions/DawoduRahmon/htmlForms/contact.html:13]: Check here [https://www.w3schools.com/html/html_forms_attributes.asp](https://www.w3schools.com/html/html_forms_attributes.asp)
@Dawodurahmonn

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**